tensorboard/plugins/pr_curve/pr_curves_plugin.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Could you add an `http_api.md`? (Copy-paste and modify one of the other plugins'.)